main-unix.c

```diff
@@ -257,14 +293,21 @@ main (int argc,
 
       flags |= JERRY_FLAG_PARSE_ONLY;
 
-      if (save_snapshot_file_name_p == NULL
-          && ++i < argc)
+      if (save_snapshot_file_name_p != NULL)
+      {
+        JERRY_ERROR_MSG ("Error: snapshot file name already specified\n");
+        print_usage (argv);
+        return JERRY_STANDALONE_EXIT_CODE_FAIL;
+      }
+
+      if (++i < argc)
       {
         save_snapshot_file_name_p = argv[i];
       }
       else
       {
-        JERRY_ERROR_MSG ("Error: wrong format of the arguments\n");
+        JERRY_ERROR_MSG ("Error: no file specified for %s\n", argv[i-1]);
```

Comment:
Oh, my bad. We are using spaces around operators. So, it's `i - 1`. (I know, I was the one who wrote `i-1` in the review. Sorry :( )